app/assets/javascripts/discourse/app/lib/langauge.js

```diff
@@ -0,0 +1,17 @@
+import I18n from "discourse-i18n";
+
+export function humanizeList(listItems) {
```

Comment:
I think rather than adding a new file, it would be OK to add this to `app/assets/javascripts/discourse/app/lib/text.js` or possibly `app/assets/javascripts/discourse/app/lib/formatter.js`